user-service/controller/user-controller.js

```diff
@@ -71,21 +80,8 @@ export async function loginUser(req, res) {
 
 export async function logoutUser(req, res) {
     try {
-        const { username } = req.body;
-        // Check for missing fields
-        if (!username) {
-            return res.status(400).json({message: 'Username is missing!'});
-        }
-        // Get token from request
-        const token = getTokenFrom(req);
-        if (!token) {
-            return res.status(401).json({message: 'Token is missing or invalid!'});
-        }
-        // Validate token
-        const tokenValid = await validateToken(username, token);
-        if (!tokenValid) {
-            return res.status(401).json({message: 'Token is missing or invalid!'});
-        }
+        const username = req.user.username;
+        const token = req.token;
         // Blacklist token when user logs out
         const resp = await _createBlacklist(token);

```

Comment:
Consider renaming `resp` since it can conflict with `res`

Reply:
Solved